src/vivarium_census_prl_synth_pop/results_processing/addresses.py

```diff
@@ -145,3 +148,29 @@ def get_household_address_map(
         address_map[obs_column] = address_data[obs_column]
 
     return address_map
+
+
+def get_city_map(
+    column_name: str,
+    obs_data: pd.DataFrame,
+    artifact: Artifact,
+    randomness: RandomnessStream,
+) -> Dict[str, pd.Series]:
+
+    # Load addresses data from artifact
+    addresses = artifact.load(data_keys.SYNTHETIC_DATA.ADDRESSES).reset_index()
+    # Get observer data to map
+    output_cols = [column_name, "state"]
+    city_data = obs_data.reset_index()[output_cols].drop_duplicates().set_index("address_id")
+
+    for state in city_data["state"].str.lower().unique():
+        cities = vectorized_choice(
+            options=addresses.loc[addresses["Province"] == state, "Municipality"],
```

Comment:
Does this mean each city has an equal chance of getting chosen?

Reply:
I see - addresses is actually ACS data and so larger cities will have more rows and thus more likely to be sampled

Reply:
Not to pick nits, but it's not ACS data -- (publicly released) ACS data does not include addresses. This is standalone address data, which comes from `libpostal` and ultimately originates from crowdsourced OpenStreetMap addresses and government records.

Reply:
But your larger point is correct; even though we have no real guarantees given the data sources that what we have is a true distribution of cities within a state, we consider it close enough (certainly better than a uniform random sample of city) so we allow the data to dictate which cities are more common.

Reply:
The documentation for this is here: https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_census_synthdata/concept_model.html#id34. It could probably be a little more explicit on the reasoning for this behavior.